db/r2dec

```diff
@@ -0,0 +1,21 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/wargio/r2dec-js"
+R2PM_DESC "[r2-r2pipe-node] an Experimental Decompiler"
+BINDIR="${R2PM_BINDIR}"
+
+R2PM_INSTALL() {
+	node --version || exit 1
+	npm install || exit 1
+	mkdir -p "${R2PM_BINDIR}" || exit 1
+	echo '#!/bin/sh' > "${R2PM_BINDIR}/r2dec" || exit 1
+	echo "exec node '`pwd`'"'/r2dec.js $*' >> "${R2PM_BINDIR}/r2dec" || exit 1
```

Comment:
why not just copy the r2dec.js with #!/usr/bin/env node as hashbang instead of relaying on /bin/sh to wrap that